packages/ui/src/models/visualization/flows/support/camel-component-schema.service.ts

```diff
@@ -141,11 +146,15 @@ export class CamelComponentSchemaService {
    */
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   private static getCamelElement(processorName: keyof ProcessorDefinition, definition: any): ICamelElementLookupResult {
+    if (!isDefined(definition)) {
```

Comment:
If there's no node definition, it's probably that the node no longer exists, so better to fallback to the `processorName`